collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2888,6 +2908,12 @@ private void updateFieldListQuestions(FormIndex lastChangedIndex) {
         }
     }
 
+    // If an answer has changed after saving one of previous answers that means it has been recalculated automatically
+    private boolean isQuestionRecalculated(FormEntryPrompt mutableQuestionBeforeSave, ImmutableDisplayableQuestion immutableQuestionBeforeSave) {
+        return !(mutableQuestionBeforeSave.getAnswerText() == null && immutableQuestionBeforeSave.getAnswerText() == null
```

Comment:
This is exactly what `Objects.equals` does, right? I was suggesting not even introducing a custom method. I don't feel strongly about it, though, if you like this better.